packages/kbn-content-management-utils/src/types.ts

```diff
@@ -161,6 +161,8 @@ export interface SavedObjectSearchOptions {
 
 /** Saved Object update options  - Pick and Omit to customize */
 export interface SavedObjectUpdateOptions<Attributes = unknown> {
+  /** Overwrite existing documents (defaults to false) */
```

Comment:
This is used by all visualization types but maps right now. I had to introduce it to `maps` in order to make it work within the metadata editing in visualize Listing page.
Other visualizations were using the `CreateOptions` type on Update in order to validate/pass the `overwrite` flag.
I can see how this can be removed, but I would like to move it in a follow up PR unless it's strictly required.

Reply:
here is some context where a related bug in Lens was fixed https://github.com/elastic/kibana/issues/160116#issuecomment-1602328308

Reply:
I'd like a more complete explanation of this which might be best done over zoom since its modifying api types.